src/ops/constant.rs

```diff
@@ -213,33 +156,33 @@ mod test {
 
     #[test]
     fn test_bad_predicate() {
-        let pred_rows = vec![type_row![Type::i64(), CLASSIC_T], type_row![]];
+        let pred_rows = [type_row![EQ_T, CLASSIC_T], type_row![]];
 
-        let res = Const::predicate(0, Value::sequence(&[]), pred_rows);
+        let res = Const::predicate(0, Value::tuple([]), pred_rows);
         assert_matches!(res, Err(ConstTypeError::TupleWrongLength));
     }
 
     #[test]
     fn test_constant_values() {
         const T_INT: Type = Type::usize();
-        const V_INT: Value = Value::Hashable(HashableValue::Int(257));
-        V_INT.check_type(&T_INT).unwrap();
-        custom_value(17.4).check_type(&CLASSIC_T).unwrap();
+        const V_INT: Value = CustomTestValue(Some(TypeBound::Eq)).into();
```

Comment:
yes these are not actually constants so won't compile, left for a future commit!